modules/qtwidgets/src/tf/tfeditor.cpp

```diff
@@ -494,39 +589,107 @@ void TFEditor::contextMenuEvent(QGraphicsSceneContextMenuEvent* e) {
                 removeControlPoint(elem);
             }
         });
+
+        connect(clearTF, &QAction::triggered, this, [this]() {
+            NetworkLock lock(tfPropertyPtr_->getProperty());
+            for (auto& elem : tfSets_) {
+                elem->clear();
+            }
+        });
+        connect(resetTF, &QAction::triggered, this, [this]() {
+            NetworkLock lock(tfPropertyPtr_->getProperty());
+            tfPropertyPtr_->getProperty()->resetToDefaultState();
+        });
     }
 
     auto transformMenu = menu.addMenu("Trans&form");
     menu.addSeparator();
 
     {
-        auto flip = transformMenu->addAction("&Horizontal Flip");
-        auto interpolate = transformMenu->addAction("&Interpolate Alpha");
-        auto equalize = transformMenu->addAction("&Equalize Alpha");
+        auto distributeAlphaEvenly = transformMenu->addAction("&Distribute Alpha Evenly");
```

Comment:
Would be nice to have buttons for the different available interactions

Reply:
that could be added I guess, but there are 10 different transforms, where would you put them all?

Reply:
Could use a dropdown button